README.md

```diff
@@ -1,2 +1,89 @@
-# DumE
-Open Projects 2021
+
+
+## DumE
+
+Open Projects 2021
+![DumE](https://github.com/rodion0917/Random/blob/main/Dum-E%20in%20Action%20-%20Green%20Object.png)

```

Comment:
Insert a line break after 'Open Projects 2021'. Also, center align all the images using the `<p>` paragraph tag and `align` attribute. For eg:
`<p align="center">`
  `<img src="https://github.com........." alt="LED Matrix">`
  `<i>LED Matrix</i>`
`</p>`

And add appropriate title for each image.
Also try to resize the images by assigning proper dimensions.

Refer this README for more info: https://raw.githubusercontent.com/marsiitr/Rube-Goldberg-Machine/master/README.md